operations/_scripts/generate/generate_tf_vars.sh

```diff
@@ -22,74 +14,266 @@ else
 fi
 
 GITHUB_IDENTIFIER="$($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh)"
+GITHUB_IDENTIFIER=$(echo $GITHUB_IDENTIFIER | tr "_" "-")
 echo "GITHUB_IDENTIFIER: [$GITHUB_IDENTIFIER]"
 
 GITHUB_IDENTIFIER_SS="$($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier_supershort.sh)"
+GITHUB_IDENTIFIER_SS=$(echo $GITHUB_IDENTIFIER_SS | tr "_" "-")
 echo "GITHUB_IDENTIFIER SS: [$GITHUB_IDENTIFIER_SS]"
 
+
+# -------------------------------------------------- #
+domain_name=
+if [ -z "$DOMAIN_NAME" ]; then
+  domain_name="domain_name = \"${DOMAIN_NAME}\""
+fi
```

Comment:
this says "if `DOMAIN_NAME` is empty, set `domain_name` to its value, right?

Is that intended?

Reply:
That's a question for @LeoDiazL 
I didn't change the logic, just the variable+usage

Reply:
It's fixed in RDS branch @mickmcgrath13 @PhillypHenning 
the variable is called ... sub_domain_name, not sub_domain

Reply:
cc @LeoDiazL

Reply:
Shouldn't it be -n instead of -z ?  @PhillypHenning